### PR TITLE
feat: contentful video and images

### PR DESCRIPTION
**Pull request by owenmerry**

### CODE REVIEW

**Comment by owenmerry** [comment]
Lets do this PR after the other goes live as its very all the same code as the other so the diff will be confusing till the first goes live

File: packages/website/src/components/nav-bar.jsx [hunk 1]
```diff
@@ -142,12 +142,12 @@ class NavBarWrapper extends Component {
     user: PropTypes.object,
     signOut: PropTypes.func.isRequired,
     location: PropTypes.object.isRequired
-  }
+  };
```
Comment (achingbrain) [inline packages/website/src/components/nav-bar.jsx:145]:
```suggestion
  }
```

File: packages/website/src/components/nav-bar.jsx [hunk 2]
```diff
@@ -142,12 +142,12 @@ class NavBarWrapper extends Component {
...
 
   state = {
     bgOpacity: 0,
     menuOpen: false
-  }
+  };
```
Comment (achingbrain) [inline packages/website/src/components/nav-bar.jsx:150]:
```suggestion
  }
```

File: packages/website/src/components/nav-bar.jsx [hunk 3]
```diff
@@ -163,19 +163,19 @@ class NavBarWrapper extends Component {
     this.setState({
       bgOpacity: pageYOffset < 100 ? pageYOffset / 100 : 1
     })
-  }
+  };
```
Comment (achingbrain) [inline packages/website/src/components/nav-bar.jsx:166]:
```suggestion
  }
```

File: packages/website/src/components/nav-bar.jsx [hunk 4]
```diff
@@ -163,19 +163,19 @@ class NavBarWrapper extends Component {
...
 
   handleToggleDropDownMenu = () => {
-    this.setState(s => ({
+    this.setState((s) => ({
```
Comment (achingbrain) [inline packages/website/src/components/nav-bar.jsx:169]:
```suggestion
    this.setState(s => ({
```

File: packages/website/src/components/nav-bar.jsx [hunk 5]
```diff
@@ -163,19 +163,19 @@ class NavBarWrapper extends Component {
...
       menuOpen: !s.menuOpen
     }))
-  }
+  };
```
Comment (achingbrain) [inline packages/website/src/components/nav-bar.jsx:172]:
```suggestion
  }
```

File: packages/website/src/components/nav-bar.jsx [hunk 6]
```diff
@@ -163,19 +163,19 @@ class NavBarWrapper extends Component {
...
 
   handleToggleDropDownUserMenu = () => {
-    this.setState(s => ({
+    this.setState((s) => ({
```
Comment (achingbrain) [inline packages/website/src/components/nav-bar.jsx:175]:
```suggestion
    this.setState(s => ({
```

File: packages/website/src/components/nav-bar.jsx [hunk 7]
```diff
@@ -163,19 +163,19 @@ class NavBarWrapper extends Component {
...
       userMenuOpen: !s.userMenuOpen
     }))
-  }
+  };
```
Comment (achingbrain) [inline packages/website/src/components/nav-bar.jsx:178]:
```suggestion
  }
```

File: packages/website/src/components/nav-bar.jsx [hunk 8]
```diff
@@ -192,7 +192,7 @@ class NavBarWrapper extends Component {
     } catch (err) {
       console.error(err)
     }
-  }
+  };
```
Comment (achingbrain) [inline packages/website/src/components/nav-bar.jsx:195]:
```suggestion
  }
```

**Review by achingbrain — APPROVED** [review]

**Pull request merged**